packages/pricing/src/services/pricing-module.ts

```diff
@@ -62,7 +62,7 @@ export default class PricingModuleService<
   @InjectManager("baseRepository_")
   async calculatePrices(
     priceSetIds: string[],
-    pricingContext: PricingContext,
+    pricingContext: PricingContext = {},
```

Comment:
comment l112 should we serialize instead?

Reply:
doesn't seem to work with non mikroorm entities